### DISCUSSION TITLE
Workflow runs overriding?

### DISCUSSION THREAD

**Discussion by inwardmovement** [community/community#25102] [Actions]
<p>When a workflow is triggered when one intance of it is already running, is the already running one cancled in favor of the new one?</p>
<p>It would be great to have an “Auto Cancellation” just like Travis:</p>
<p>“<em>Auto Cancellation allows you to only run builds for the latest commits in the queue.</em>”</p>

**Comment by weide-zhou** [comment, marked as answer]
<p>Thanks <a class="mention" href="/inwardmovement">@inwardmovement</a>.</p>
<p>Currently It’s not supported to cancel/override the existing running workflow in favor of new commits, but this has been ALREADY raised as a feature request which github is working on.</p>
<p>I don’t have any timeline to share yet, it’s recommeded to check changelog here for any changes avaliable: <a href="https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.blog%2Fchangelog%2F&amp;data=02%7C01%7Cv-wadzho%40microsoft.com%7C6a5bb7bffc7c46295ac508d789d78aed%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637129431147486429&amp;sdata=6prn6iPFCd1uvz63FWvlcLiDwcaOIs5mtNr4nTe5Tuk%3D&amp;reserved=0" rel="nofollow noopener">https://github.blog/changelog/</a>.</p>
